backend/business-partner-agent/src/main/java/org/hyperledger/bpa/impl/aries/ProofManager.java

```diff
@@ -84,6 +93,21 @@
     @Inject
     MessageService messageService;
 
+    public List<BPAPresentationExchange> getProofRequests(Optional<UUID> partnerId,
+            @Nullable ProofRequestsRequest req) {
+        List<BPAPresentationExchange> result = new ArrayList<>();
+        try {
+            partnerId.ifPresentOrElse((pId) -> {
+                peRepo.findByPartnerId(pId).forEach(result::add);
+            }, () -> {
+                peRepo.findAll().forEach(result::add);
+            });
+        } catch (EntityNotFoundException e) {
```

Comment:
Does this happen or is an empty list returned?

Reply:
outdated. Grouped together with all PartnerProof and FE partitions.